site2/docs/reference-pulsar-admin.md

```diff
@@ -2333,17 +2342,252 @@ Remove the message TTL for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-max-consumers tenant/namespace/topic
+```
+
+### `set-max-consumers`
+Set max-Consumers for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-max-consumers tenant/namespace/topic
+```
+
+### `remove-max-consumers`
+Remove max-Consumers for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-max-consumers tenant/namespace/topic
+```
+
+### `get-max-unacked-messages-per-subscription`
+Get maxUnackedMessagesPerSubscription for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-max-unacked-messages-per-subscription tenant/namespace/topic
+```
+
+### `set-max-unacked-messages-per-subscription`
+Set maxUnackedMessagesPerSubscription for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-max-unacked-messages-per-subscription tenant/namespace/topic
+```
+
+### `remove-max-unacked-messages-per-subscription`
+Remove maxUnackedMessagesPerSubscription for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-max-unacked-messages-per-subscription tenant/namespace/topic
+```
+
+### `get-max-unacked-messages-per-consumer`
+Get maxUnackedMessagesPerConsumer for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-max-unacked-messages-per-consumer tenant/namespace/topic
+```
+
+### `set-max-unacked-messages-per-consumer`
+Set maxUnackedMessagesPerConsumer for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-max-unacked-messages-per-consumer tenant/namespace/topic
+```
+
+### `remove-max-unacked-messages-per-consumer`
+Remove maxUnackedMessagesPerConsumer for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-max-unacked-messages-per-consumer tenant/namespace/topic
+```
+
+### `get-compaction-threshold`
+Get compactionThreshold for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-compaction-threshold tenant/namespace/topic
+```
+
+### `set-compaction-threshold`
+Set compactionThreshold for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-compaction-threshold tenant/namespace/topic
+```
+
+### `remove-compaction-threshold`
+Remove compactionThreshold for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-compaction-threshold tenant/namespace/topic
+```
+
+### `get-offload-policies`
+Get the offload policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-offload-policies tenant/namespace/topic
+```
+
+### `set-offload-policies`
+Set the offload policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-offload-policies tenant/namespace/topic
+```
+
+### `remove-offload-policies`
+Remove the offload policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-offload-policies tenant/namespace/topic
+```
+
+### `get-inactive-topic-policies`
+Get the inactive policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-inactive-topic-policies tenant/namespace/topic
+```
+
+### `set-inactive-topic-policies`
+Set the inactive policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-inactive-topic-policies tenant/namespace/topic
```

Comment:
should document the parameters defined here: https://github.com/apache/pulsar/blob/master/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java#L1859-L1871